src/printer.js

```diff
@@ -575,49 +575,24 @@ function genericPrintNoParens(path, options, print, args) {
 
       return concat(parts);
     case "AwaitExpression":
-      parts.push("await");
-
-      if (n.all) {
```

Comment:
```js
async function f() {
  await *f();
}
```

Babylon throws with

> await* has been removed from the async functions proposal. Use Promise.all() instead. (2:2)

Flow parses it as a binary expression with the identifier `await` on the left. So fair to say we don't need it anymore.